Owncloud iOs Client/Files/UploadFromOtherApp/UploadFromOtherAppViewController.m

```diff
@@ -985,6 +986,8 @@ - (void) overWriteFile {
         //Set this file as an overwritten state
         [ManageFilesDB setFileIsDownloadState:file.idFile andState:overwriting];
         [UploadUtils updateOverwritenFile:file FromPath:_filePath];
+        ManageThumbnails *manageThumbnails = [ManageThumbnails sharedManager];
```

Comment:
We should use directly the singleton not getting the singleton in a local instance. Just to use the same style on all the app (like the other singletons) :wink: